src/main/scala/no/ndla/searchapi/controller/SearchController.scala

```diff
@@ -242,151 +226,9 @@ trait SearchController {
-    get("/article/", operation(articleSearchDoc)) {
-      val query = paramOrNone(this.query.paramName)
-      val sort = Sort.valueOf(paramOrDefault(this.sort.paramName, ""))
-      val language = paramOrDefault(this.language.paramName, Language.AllLanguages)
-      val license = paramOrNone(this.license.paramName)
-      val pageSize = intOrDefault(this.pageSize.paramName, SearchApiProperties.DefaultPageSize)
-      val page = intOrDefault(this.pageNo.paramName, 1)
-      val idList = paramAsListOfLong(this.learningResourceIds.paramName)
-      val articleTypesFilter = paramAsListOfString(this.contextTypes.paramName)
-      val fallback = booleanOrDefault(this.fallback.paramName, default = false)
-
-      articleSearch(query, sort, language, license, page, pageSize, idList, articleTypesFilter, fallback)
-    }
-
-    private def learningPathSearch(query: Option[String],
-                                   withIdIn: List[Long],
-                                   taggedWith: Option[String],
-                                   sort: Option[Sort.Value],
-                                   language: String,
-                                   page: Int,
-                                   pageSize: Int,
-                                   fallback: Boolean) = {
-
-      val result = query match {
-        case Some(q) =>
-          learningPathSearchService.matchingQuery(
-            query = q,
-            withIdIn = withIdIn,
-            taggedWith = taggedWith,
-            language = language,
-            sort = sort.getOrElse(Sort.ByRelevanceDesc),
-            page = page,
-            pageSize = pageSize,
-            fallback = fallback
-          )
-        case None =>
-          learningPathSearchService.all(
-            withIdIn = withIdIn,
-            taggedWith = taggedWith,
-            language = language,
-            sort = sort.getOrElse(Sort.ByTitleAsc),
-            page = page,
-            pageSize = pageSize,
-            fallback = fallback
-          )
-      }
-
-      result match {
-        case Success(searchResult) => searchResult
-        case Failure(ex)           => errorHandler(ex)
-      }
-    }
-
-    private val learningPathSearchDoc = (apiOperation[SearchResult[LearningPathSummary]]("getLearningpaths")
-      summary "Find learningpaths"
-      notes "Shows all learningpaths. You can search too."
-      parameters (
-        asHeaderParam[Option[String]](correlationId),
-        asQueryParam[Option[String]](query),
-        asQueryParam[Option[String]](sort),
-        asQueryParam[Option[String]](language),
-        asQueryParam[Option[Int]](pageSize),
-        asQueryParam[Option[Int]](pageNo),
-        asQueryParam[Option[Long]](learningResourceIds),
-        asQueryParam[Option[Boolean]](fallback),
-        asQueryParam[Option[String]](tag)
-    )
-      authorizations "oauth2"
-      responseMessages response500)
-    get("/learningpath/", operation(learningPathSearchDoc)) {
```

Comment:
Fjernes alle disse fordi de ikke brukes?

Reply:
Yap, også er det samme søk som i de andre api'ene (uten taksonomi).
Så da slipper vi i alle fall å vedlikeholde selve søket to steder selvom vi fortsatt må vedlikeholde søketing to steder :)